tests/testthat/test-lomax.R

```diff
@@ -85,9 +85,9 @@ test_that("custom-lomax", {
 
   # Check of brms can fit the custom family and recover the intercept and shape
   expect_brms_family(
-    intercept = 5,
```

Comment:
Why did you change this?

Reply:
Yeah, that change is probably not necassary. Just should decrease failures of this specific test. We can revert this, if you prefer.

Reply:
I guess, Lomax brms is the one test, I observed failing the most often. If I had to guess: 1/5 times I hit devtools-test, the expect_brms of test-lomax fails. Maybe we should think about relaxing this specific test somehow.